Dockerfile.metering-ansible-operator

```diff
@@ -1,34 +1,27 @@
 # need the helm-cli from the helm image
 FROM quay.io/openshift/origin-metering-helm:latest as helm
 # final image needs kubectl, so we copy `oc` from cli image to use as kubectl.
-FROM openshift/origin-cli:latest as cli
+FROM registry.svc.ci.openshift.org/ocp/4.6:cli as cli
 # the base image is the ansible-operator's origin images
-FROM quay.io/openshift/origin-ansible-operator:4.6
+FROM registry.svc.ci.openshift.org/ocp/4.6:ansible-operator
 
 USER root
-RUN set -x; INSTALL_PKGS="curl bash ca-certificates less which inotify-tools openssl" \
-    && yum clean all && rm -rf /var/cache/yum/* \
-    && yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm  \
-    && yum install --setopt=skip_missing_names_on_install=False -y \
-        $INSTALL_PKGS  \
-    && yum clean all \
-    && rm -rf /var/cache/yum
-
-ENV TINI_VERSION v0.18.0
-ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
-RUN chmod +x /tini
+RUN set -x;
+ARG INSTALL_PKGS="less python2-botocore python2-boto3 openssl tini"
+
+# the boto core packages will only install if the following repo are added w/out removing base repos
+RUN curl http://base-4-4.ocp.svc > /etc/yum.repos.d/base-4-4.repo
```

Comment:
Using the proxied yum repos via these links promises to make same packages available as on ART build.

We must add this repo to make the boto packages available.

Reply:
Are there ways to run this image locally?

Reply:
there's no existing base image you can use that already has this repo?  can we change the ansible-operator image you're building on top of to properly include it?

Reply:
@timflannagan1 well I'm sure you can grab the artifact and run that, but as far as building no, it's the major slowness of this work, the proxies are only reachable from the CI build.

@bparees We could switch off of ansible-operator as a base, there might be a lot to additionally do at that point. There might be a better ansible-operator image, like ansible-operator-aws, or something, but how to inspect the available images? How would we change  ansible-operator to give us what we want, would we fork ansible-operator and then maintain that fork?

Reply:
actually i think the best way to do this is to do an intermediary image, again as part of your CI release config, see:

https://github.com/openshift/release/blob/master/ci-operator/config/openshift-priv/images/openshift-priv-images-release-4.4.yaml#L39-L45

as an example.

then use the tag that step produces, as your base image in the job config/substitution logic.

that way your dockerfile doesn't need to contain CI specific logic and you can more easily collapse on a single dockerfile for CI and OSBS

Reply:
Actually since .../ocp/4.4:base tag exists, committed a change that makes it a layer and copies from it, curl'd repo is eliminated.